### ISSUE TITLE
Account google (kalendar, Korganizer, widget)  nothing is displayed.

### ISSUE THREAD

**Issue by carabao72** [ZaWertun/fedora-copr-kde5#98]
Hello,

I can no longer display my google calendar in Korganizer, Kalendar or even in the "clock" widget. If the configuration goes well: addition, authorization, choice of calendars, after nothing is displayed.
I already deleted all the "google" account and restarted the procedure without further success.

Fedora 35, Plasma 5.24.1

**Comment by ZaWertun** [comment]
You're right... Just checked - same happens to me, no calendar events anymore.

**Comment by carabao72** [comment]
Thank You

**Comment by ZaWertun** [comment]
Could you check your version of `kdepim-runtime ` package? (`rpm -q kdepim-runtime `).

**Comment by ZaWertun** [comment]
It looks like you have `kdepim-runtime` installed from official Fedora package
and for Fedora 35 there is no build with this patch: https://github.com/ZaWertun/fedora-copr-kde5/blob/master/APPS/kdepim-runtime/kdepim-runtime-21.12.1-disable-google-contacts-sync.patch (or it's analog).

I'll rebuild package => issue should be gone after update.
For me calendars is back after clicking "Restart" on account in Kalendar settings (Calendar Sources).

**Comment by carabao72** [comment]
```
rpm -q kdepim-runtime
kdepim-runtime-21.12.2-2.fc35.x86_64
```

**Comment by g6avk** [comment]
FWIW I can confirm Korg works fine with kdepim-runtime-21.12.2-1.fc35.x86_64 from your repo. I try not to mix the official and your repo.

**Comment by ZaWertun** [comment]
> ```
> rpm -q kdepim-runtime
> kdepim-runtime-21.12.2-2.fc35.x86_64
> ```

I need 1 more release bump then.

**Comment by carabao72** [comment]
```
 rpm -q kdepim-runtime
kdepim-runtime-21.12.2-3.fc35.x86_64
rpm -q kalendar
kalendar-1.0.0-1.fc35.x86_64
```
After clicking on "restart", the calendars are displayed

Thank You

**Comment by carabao72** [comment]
With the "digital clock 3.0" widget, calendars are still not displayed but this problem is already older. When my PC comes out of sleep, calendars no longer appear, I have to restart the PC to see them again.

But this is another problem, I think?

**Comment by ZaWertun** [comment]
> With the "digital clock 3.0" widget, calendars are still not displayed but this problem is already older. When my PC comes out of sleep, calendars no longer appear, I have to restart the PC to see them again.
> 
> But this is another problem, I think?

IDK, calendars should work now.
Maybe reboot was needed so that plasma widget could "see" calendars.
